cmd/registry/cmd/resolve/resolve.go

```diff
@@ -64,19 +64,14 @@ func Command(ctx context.Context) *cobra.Command {
 
 			log.Printf("Generated %d actions. Starting Execution...", len(actions))
 
-			taskQueue := make(chan core.Task, 1024)
-			for i := 0; i < 64; i++ {
-				core.WaitGroup().Add(1)
-				go core.Worker(ctx, taskQueue)
-			}
-			defer core.WaitGroup().Wait()
-			defer close(taskQueue)
-
 			// Submit tasks to taskQueue
```

Comment:
Is this comment still relevant? It seems that this change is removing the task queue. Is this intentionally now serializing task execution? Is that temporary?

Reply:
Yeah the comment was not relevant. Serial execution is a temporary change. We currently can't use task queues because of this issue https://github.com/apigee/registry/issues/239. Once that is resolved, the plan is to switch back to task queue.

Reply:
OK. SGTM!